reference/6/Microsoft.PowerShell.Core/About/about_Functions_Advanced_Parameters.md

```diff
@@ -39,10 +39,10 @@ has the following characteristics:
 - It takes input from the pipeline.
 - It takes an array of strings as input.
 
-```powershell
+```PowerShell
```

Comment:
Do not camel-case code fence tags. Please fix all instances.

Reply:
It is Pascal case and also everywhere in Azure and Azure Stack docs we do it that way...

https://raw.githubusercontent.com/MicrosoftDocs/azure-docs/master/articles/azure-stack/azure-stack-powershell-install.md